test/lightning_web/live/work_order_live_test.exs

```diff
@@ -46,35 +46,25 @@ defmodule LightningWeb.RunWorkOrderTest do
     test "WorkOrderComponent", %{
       project: project
     } do
-      workflow = insert(:workflow, project: project)
-      trigger = insert(:trigger, type: :webhook, workflow: workflow)
-      job = insert(:job, workflow: workflow)
+      %{jobs: [job]} = workflow = insert(:simple_workflow, project: project)
 
       dataclip = insert(:dataclip)
 
-      reason =
-        insert(:reason,
-          type: :webhook,
-          trigger: trigger,
-          dataclip: dataclip
-        )
-
```

Comment:
Reasons are now deprecated and 'conceptually' exist on the WorkOrder or Attempt directly.

In the case of WorkOrder, there are now `trigger` and `dataclip` fields.

We must hunt down all mentions of reason, probably best starting with this file.